lnwallet/reservation.go

```diff
@@ -98,10 +98,6 @@ type ChannelReservation struct {
 	// fundingTx is the funding transaction for this pending channel.
 	fundingTx *wire.MsgTx
 
-	// For CLTV it is nLockTime, for CSV it's nSequence, for segwit it's
```

Comment:
Nice, happy to see this removed. This was left over from when we were toying around with a CLTV based channel construction (assuming there isn't a malleability fix).